src/TipTapEditor/CollapsibleBlock/nodes/CollapsibleBlockNode.ts

```diff
@@ -25,15 +25,14 @@ export const CollapsibleBlockNode = Node.create({
 
   addKeyboardShortcuts() {
     return {
-      'Mod-Enter': () => {
-        return this.editor
+      'Mod-Enter': () =>
+        this.editor
```

Comment:
This is the `arrow-body-style` rule.

Reply:
Don't you prefer `block body` for arrow functions when it has a line break? I find it easier to read, and I always think about a well-known security issue a few years ago because an if/else didn't have curly braces and someone added another instruction (so, another line) which would run every time (regardless of the condition)

Reply:
I don't mind this, actually! The if/else continuation line issue is very real, which is why I think `curly` is a good idea. But since the thing on the RHS of `=>` is an expression, not a statement, I think there's less room for trouble. In most cases if you added an extra statement after an arrow function with a line break, it would be a syntax error or create dead code (which TypeScript would complain about).